src/docker-extract.c

```diff
@@ -255,6 +256,21 @@ int extract_tar(const char *tarfile, const char *rootfs_dir) {
             continue;
         }
 
+        // Issue 977 - Force write perms needed for user builds
```

Comment:
For clarity, it probably would have been 'better' to define a macro perhaps in the include file, to deal with the conditional compilation of that code, but that work.